api/lib/domain/events/handle-auto-jury.js

```diff
@@ -101,7 +101,7 @@ async function _handleAutoJuryV2({
 }
 
 function _areV3CertificationCourses(certificationCourses) {
-  return certificationCourses[0].getVersion() === CertificationVersion.V3;
+  return CertificationVersion.isV3(certificationCourses[0].getVersion());
```

Comment:
je veux pas tout remettre en cause mais pourquoi pas 
```suggestion
  return certificationCourses[0].isV3();
```

Reply:
Ca peut être une autre amélioration indeed.